instagram/instagram/src/Components/App.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
```

Comment:
2 App components? one in Components and another in src? why do you have it set up this way?